### ISSUE TITLE
When Scroll Page: theme.js:4 Uncaught TypeError: Cannot read property 'split' of undefined

### ISSUE THREAD

**Issue by aimer1124** [GitbookIO/gitbook#1525]
When i scroll [my page ](https://aimer1124.gitbooks.io/jmeter-tutorial/content/chapter1/%E4%BA%86%E8%A7%A3%E5%B9%B6%E5%90%AF%E5%8A%A8JMeter.html)  in my Gitbook Content,the browser console is something wrong.Like this

```
theme.js:4 Uncaught TypeError: Cannot read property 'split' of undefined
```

![console](https://cloud.githubusercontent.com/assets/5865980/18347430/02843d86-75f8-11e6-9e39-b45978259f88.png)

And when i scroll, the wrong number increase fast.The background color is black

![background](https://cloud.githubusercontent.com/assets/5865980/18771879/1977aac2-8175-11e6-8ad8-99edb01aecf7.png)

---

Gitbook and node Version

```
➜  jmeter-tutorial git:(master) ✗ gitbook --version
CLI version: 2.3.0
GitBook version: 3.2.0
➜  jmeter-tutorial git:(master) ✗ node -v
v4.4.3
```

My PC Version

```
ProductName:    Mac OS X
ProductVersion: 10.11.2
BuildVersion:   15C50
Chrome: 53.0.2785.116 (64-bit)
```

This is my gitbook [homepage](https://aimer1124.gitbooks.io/jmeter-tutorial/content),but when i use Firefox browser,everything goes ok.

**Comment by maxkoryukov** [comment]
I have the very similar error:

```
theme.js:12924 Uncaught TypeError: Cannot read property 'top' of undefined
```

I have not-uglified line number (12924), just because I forced **gitbook** to use not-compressed **default-theme**  (see also GitbookIO/theme-default#43)

So, in my case the errors was caused by the non-existent _bookmark_ in the `SUMMARY.md`. Lets say:

* in `SUMMARY.md` I have a link like
    ```markdown
    * [about section](page.md#about)
    ```
* but my `page.md` does not contain a chapter named `about`. Even if I have an explicit
    ```html
    <a name="about"></a>
    ```
    this does not help - A-tag will not be recognized by name;

The error occurs in the `GitbookIO/theme-default` package , in the file `/src/js/theme/navigation.js`, line  **63**, function `getElementTopPosition`. In the following code:

```js
dest = $el.position().top;
```

`$el` doesn't contain any tag, `$el.position()` returns undefined.

**Comment by maxkoryukov** [comment]
Actually, this issue should be moved to the `default-theme` repo

**Comment by SamyPesse** [comment]
This issue can stay in this repository, since we moved to one repository for all [packages](https://github.com/GitbookIO/gitbook/tree/master/packages) now.

I'll close this issue once v4 is released as stable.

**Comment by maxkoryukov** [comment]
Fixed in fd742a15ffcd07e881d9888ac15ed1dbeebfcf5a

**Comment by maxkoryukov** [comment]
Fixed in https://github.com/GitbookIO/theme-default/commit/fd742a15ffcd07e881d9888ac15ed1dbeebfcf5a

**Comment by maxkoryukov** [comment]
ping @SamyPesse @AaronO  to close this issue - it is closed, I've attached a link to a commit with a patch

**Comment by morsko1** [comment]
hi @maxkoryukov,
you commented that this issue fixed by [GitbookIO/theme-default@fd742a1](https://github.com/GitbookIO/theme-default/commit/fd742a15ffcd07e881d9888ac15ed1dbeebfcf5a)

but in my case it didn`t solve issue of console errors when scrolling page:
![image 049](https://user-images.githubusercontent.com/22314707/44772092-9d193b80-ab75-11e8-9eb6-682804033ffb.png)

if i use your fix of [closed PR](https://github.com/GitbookIO/theme-default/pull/44) - there are no errors in console.

so can you confirm that [GitbookIO/theme-default@fd742a1](https://github.com/GitbookIO/theme-default/commit/fd742a15ffcd07e881d9888ac15ed1dbeebfcf5a) fixed the console errors when scrolling?

and if it was not fixed by [GitbookIO/theme-default@fd742a1](https://github.com/GitbookIO/theme-default/commit/fd742a15ffcd07e881d9888ac15ed1dbeebfcf5a) - could you reopen your [closed PR](https://github.com/GitbookIO/theme-default/pull/44) please?

thanks

#https://github.com/GitbookIO/theme-default/pull/44
